pkg/cluster/initialize.go

```diff
@@ -61,18 +61,35 @@ func (u *simpleUpgrader) Initialize(ctx context.Context, cs *api.OpenShiftManage
 	return b.CreateBlockBlobFromReader(bytes.NewReader(csj), nil)
 }
 
-func (u *simpleUpgrader) CreateConfigStorageAccount(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
+func (u *simpleUpgrader) createLRSStorageAccount(ctx context.Context, resourceGroup, accountName, location string, tags map[string]*string) error {
 	parameters := azstorage.AccountCreateParameters{
 		Sku: &azstorage.Sku{
 			Name: azstorage.StandardLRS,
 		},
 		Kind:     azstorage.Storage,
-		Location: &cs.Location,
+		Location: &location,
+		Tags:     tags,
+	}
+	return u.accountsClient.Create(ctx, resourceGroup, accountName, parameters)
+}
+
+func (u *simpleUpgrader) CreateAzureFileStorageAccount(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
```

Comment:
this should be being created in the ARM template - look at how the registry storage account is populated and follow that

Reply:
good catch, this got leftover from when I was testing with static provisioning and had to create the share beforehand, going to use ARM instead.